src/openforms/logging/tests/test_events.py

```diff
@@ -33,3 +38,42 @@ def test_submission_start(self):
 
             log = TimelineLogProxy.objects.last()
             self.assertEqual(_("Anonymous user"), log.fmt_user)
+
+    def test_submission_logic_evaluated(self):
+        submission = SubmissionFactory.create()
+        form = FormFactory.create()
+        form_step = FormStepFactory.create(form = form)
+        submission_step = SubmissionStepFactory.create(submission = submission, form_step = form_step)
+        json_logic_trigger = {
+            ">": [
+                {
+                "date": {
+                    "var": "birthdate"
+                }
+                },
+                {
+                "date": "2022-06-20"
+                }
+            ]
+        }
+
+        actions = [{
+            "component": "firstname",
+            "formStep": "",
+            "action": {
+                "type": "property",
+                "property": {
+                "value": "disabled",
+                "type": "bool"
+                },
+                "state": True
+            }
+        }]
+        rule = FormLogic(form=form, json_logic_trigger=json_logic_trigger, actions=actions)
+        logevent.submission_logic_evaluated(submission, rule)
+        log = TimelineLogProxy.objects.filter(
+            template="logging/events/submission_logic_evaluated.txt", 
+            object_id = submission.id)[0]
+        
+        self.assertIsNotNone(log)
+
```

Comment:
Please structure your tests nicely with whitespace:

```
def test_something(self):
    ...  # test data setup

    result = call_thing_your_are_testing(...)

    ... # make assertions
```

having a blank line before and after the call of the thing you're testing makes the structure of `setup / call / assert` more obvious and easier to understand/debug